dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/AndroidTracerBuilder.kt

```diff
@@ -0,0 +1,30 @@
+package com.datadog.android.tracing.internal
+
+import com.datadog.android.core.internal.data.Writer
+import com.datadog.android.tracing.TracerBuilder
+import com.datadog.android.tracing.internal.data.TracesWriter
+import datadog.opentracing.DDSpan
+import datadog.opentracing.DDTracer
+import datadog.trace.api.Config
+import java.util.Properties
+
+internal class AndroidTracerBuilder(
+    val serviceName: String,
+    val writer: Writer<DDSpan>
+) : TracerBuilder {
+    companion object {
+        private const val SERVICE_NAME = "service.name"
```

Comment:
I know it's overwritten by `serviceName` in public API, but if at some point this evaluates to default service name I think `"android"` might be better value.

Reply:
Is not us who decide this. This is already set in the _dd-trace-ot_ API

Reply:
It's the default for dd-trace-ot but I agree, for our customers, we should have a sensible default value, and using the same as the default value for logs makes a lot of sense.

Reply:
Oh...I know why are you making the confusion...check my last commit that name was changed now you will understand. Basically that's the key name in the dd-trace-ot API Config